src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java

```diff
@@ -72,13 +83,7 @@ public static String getSlotTrigger(String slotName) {
     SLOTS_SELECTORS = new HashMap<>();
     SLOTS_SELECTORS.put(BOTTOM_LB, "#BOTTOM_LEADERBOARD");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "#BOTTOM_LEADERBOARD" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)